test_cases/abi_routing/contract.py

```diff
@@ -37,10 +38,10 @@ def noop_with_uint64(self, a: arc4.UInt64) -> arc4.UInt8:
     @arc4.abimethod(
         allow_actions=[
             "NoOp",
-            "OptIn",
+            OnCompleteAction.OptIn,
             "CloseOut",
-            "UpdateApplication",
-            "DeleteApplication",
+            OnCompleteAction.UpdateApplication,
```

Comment:
Any reason for the mix of strings and enum values here? Is it to prove both work?

Reply:
Yeah just testing either work